### PR TITLE
fix: Extra space between parentDiv and childDiv

### PR DESCRIPTION
**Pull request by anuragnegi000**
**What kind of change does this PR introduce?**

bugfix

**Issue Number:**

Fixes #
#2071 
**Did you add tests for your changes?**

Not required
**Snapshots/Videos:**

Before:- 
beforep1

After:- 
latest

**If relevant, did you update the documentation?**

**Summary**

This change is motivated by the need to optimize space utilization within the parent container, which was previously taking up extra space due to the inclusion of the ProfileDropdown component. By removing this excess space, aiming to achieve a cleaner and more visually appealing layout.

**Does this PR introduce a breaking change?**

No
**Other information**
--

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

## Summary by CodeRabbit

- **New Features**
	- Enhanced visibility and interaction for the profile dropdown component with updated dimensions and positioning.

- **Bug Fixes**
	- Adjusted layout and spacing in multiple modules, potentially leading to a more streamlined user interface by removing unnecessary classes and padding.

- **Style**
	- Introduced structural improvements to the styling of the profile dropdown and User Portal components, enhancing overall usability and alignment within the UI.

### CODE REVIEW

**Comment by palisadoes** [comment]
Please fix the first comment so that each issue listed automatically closes. The PR_GUIDELINES.md file has details.

Please also fill in the template for the pull request as completely as you can. It greatly helps others searching for code changes in future and helps others understand the summarized rationale for the work you have done

**Comment by anuragnegi000** [comment]
Screenshot 2024-08-02 141926
Why all these commits are reflecting on my PR?
@palisadoes

File: src/screens/UserPortal/Events/Events.module.css [hunk 1]
```diff
@@ -14,14 +14,9 @@
   max-width: 800px;
 }
 
-.colorLight {
-  background-color: #f1f3f6;
-}
-
```
Comment (pranshugupta54) [inline src/screens/UserPortal/Events/Events.module.css:20]: Why are you removing light color from all screens?

**Review by pranshugupta54 — CHANGES_REQUESTED** [review]

**Comment by pranshugupta54** [comment]
@anuragnegi000 
Please fix linting issues, check Guidelines for more info on linters and formatters

**Comment by pranshugupta54** [comment]
> Why all these commits are reflecting on my PR?
I think you did some rebasing, you can make a fresh new branch from newly fetched develop to have no previous commits

File: src/screens/UserPortal/Events/Events.module.css (hunk 1 above)
Comment (anuragnegi000) [reply src/screens/UserPortal/Events/Events.module.css:20]: because it is making all the screens look odd

**Pull request closed without merging**